annotations/openshift-annotations/src/main/java/io/ap4k/openshift/configurator/ApplySourceToImageHook.java

```diff
@@ -23,10 +23,13 @@
 @Description("Apply source to image build hook.")
 public class ApplySourceToImageHook extends Configurator<S2iConfigFluent> {
 
+  private static final String AP4K_BUILD = "ap4k.build";
   private static final String AP4K_DEPLOY = "ap4k.deploy";
 
   @Override
   public void visit(S2iConfigFluent config) {
-    config.withAutoDeployEnabled(Boolean.parseBoolean(System.getProperty(AP4K_DEPLOY, String.valueOf(config.isAutoDeployEnabled()))));
+    config
+      .withAutoBuildEnabled(Boolean.parseBoolean(System.getProperty(AP4K_BUILD, String.valueOf(config.isAutoBuildEnabled()))))
+      .withAutoBuildEnabled(Boolean.parseBoolean(System.getProperty(AP4K_DEPLOY, String.valueOf(config.isAutoBuildEnabled()))));
```

Comment:
Isn't this wrong?

Reply:
I mean shouldn't be `withAutoDeployEnabled`?

Reply:
correct. let me fix that.